src/main/scala/io/smartdatalake/workflow/dataobject/CsvFileDataObject.scala

```diff
@@ -82,9 +85,9 @@ case class CsvFileDataObject( override val id: DataObjectId,
   override val fileName: String = "*.csv*"
 
   private val formatOptionsDefault = Map(
-    "header" -> "false",
+    "header" -> "true",
     "inferSchema" -> "false",
-    "delimiter" -> "|",
+    "delimiter" -> ",",
```

Comment:
I rather prefer `|` as delimiter because it occurs less often in string fields.

Reply:
hi @kaelte, but that would then be a Psv (pipe separated values) file... i suggest to use the pseudo standard for csv: delimiter = comma, header = true and override that in the config if we have other needs (like psv :-). Could you live with that?

Reply:
Yes I can live with comma as delimiter.
However a pipe delimited file is still a CSV: _Character-separated values_ .